js/pages/categoryPage/startPlay.js

```diff
@@ -0,0 +1,189 @@
+
+function showResult() {
+  const blockedLayer = document.querySelector('.play-block');
+  const result = document.createElement('img');
+  result.setAttribute('alt', 'result');
+  result.classList.add('result-img');
+  cleanPage();
+  hidePlayBtn();
+
+  if (blockedLayer.querySelector('.result-img')) {
+    blockedLayer.querySelector('.result-img').remove();
+  }
+
+  if (document.querySelector('.heart-bad')) {
+    result.setAttribute('src', './assets/icons/result-sad.jpg');
+    showTextUnderPlayBtn(`You lose. Number of mistakes = ${document.querySelectorAll('.heart-bad').length}`);
+    new Audio('./assets/audio/loseGame.mp3').play();
+  } else {
+    result.setAttribute('src', './assets/icons/win.jpg');
+    showTextUnderPlayBtn("You won! You didn't make a single mistake");
+    new Audio('./assets/audio/winGame.mp3').play();
+  }
+
+  blockedLayer.prepend(result);
+}
+function finishGame() {
+  showBlockOnPlay();
+  showResult();
+  const nav = document.querySelector('.navbar');
+  // document.querySelector('.navbar').style = 'pointer-events: none;';
+  nav.classList.add('navbar-disabled');
+  setTimeout(() => {
+    showPage('sections');
+    // document.querySelector('.navbar').style = '';
+    nav.classList.remove('navbar-disabled');
+  }, 3000);
+}
+
+function addHeart(type) {
+  const parent = document.querySelector('.hearts-container');
+  if (parent) {
+    const heart = document.createElement('img');
+
+    heart.setAttribute('alt', 'heart');
+
+    if (type === 'GOOD') {
+      heart.setAttribute('src', './assets/icons/heart.png');
+      heart.classList.add('heart-img', 'heart-good');
+    }
+    if (type === 'BAD') {
+      heart.setAttribute('src', './assets/icons/heart-bad.png');
+      heart.classList.add('heart-img', 'heart-bad');
+    }
+    parent.append(heart);
+
+    if (parent.querySelectorAll('.heart-img').length > 7) {
+      let f = 0;
+      const arr = parent.querySelectorAll('.heart-img');
+      arr.forEach((singleHeart) => {
+        if (!singleHeart.classList.contains('hide') && f === 0) {
+          singleHeart.classList.add('hide');
+          f += 1;
+        }
+      });
+    }
+  } else {
+    const header = document.querySelector('.header-main');
+
+    const divParent = document.createElement('div');
+    divParent.classList.add('hearts-container');
+
+    header.append(divParent);
+    addHeart(type);
+  }
+}
+
+function setNextCardAsActive() {
+  const resourse = JSON.parse(localStorage.getItem('statistic'));
+  const currentPage = localStorage.getItem('page');
+
+  const cards = document.querySelectorAll('.card');
+  const numOfActiveCard = getCardsOrder()[0];
+
+  const activeCard = cards[numOfActiveCard];
+  let currentCardObj;
+  let arrForSearch;
+
+  if (currentPage !== 'difficult') {
+    arrForSearch = resourse[currentPage];
+  } else {
+    arrForSearch = returnAllWords();
+  }
+  arrForSearch.forEach((i) => {
+    if (i.title.toLocaleLowerCase().replace(' ', '') === activeCard.id) {
```

Comment:
i saw it before `.toLocaleLowerCase().replace(' ', '')` probably is it worth moving the coed to the function as util?